README.md

```diff
@@ -1 +1,173 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Ruby
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Having Solver class with 3 methods
+- Factorial method to calculate factorial of a number
+- Reverse method to Reverse a word chracter
+- FizzBizz method
+- Unit test using RSpec
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+- Laptop
+- Github
+- Git
+- Ruby
+- Linters
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Install [Ruby](https://www.ruby-lang.org/en/documentation/installation/).
+- Create a repo on your repositores files.
+- Clone or make a copy of this repo on your local machine.
+- Follow GitHub flow.
+- A carefully reading of this README.md is required.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+sh
+  cd my-folder
+  git clone https://github.com/nilab-khwaja/TDD-Project.git
+```
+### Install
+
+Install this project with:
+
+```
+sh
+  bundle install
+```
+### Usage
+
+To run the project, run the following command:
+
+```
+sh
+  ruby main.rb
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+rubocop
+rspec spec
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Nilab Khwajazada**
+
+- GitHub: [@nilab-khwaja](https://github.com/nilab-khwaja)
+- Twitter: [@nilab](https://x.com/NilabKhwajazada?t=hEkqy2CCNFOIfGtbmD2gLA&s=35)
+- LinkedIn: [@nilab-khwajazada](https://www.linkedin.com/in/nilab-khwajazada)
+
+👤 **Afimia Fidelis**
+
+- GitHub: [@Afimia Fidelis](https://github.com/Izebiafe)
+- Twitter: [@Afimia Fidelis](https://x.com/Izebiafe)
+- LinkedIn: [@Afimia Fidelis](https://www.linkedin.com/in/Izebiafe)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Performance Optimization
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/nilab-khwaja/TDD-Project/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project,please give us a ⭐️!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for giving me the opportunity to learn and grow as a developer.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT] licensed.
+
```

Comment:
- [ ] The link to your **_MIT LICENSE_** was omitted in your **_README_**. It is important to include the **_MIT LICENSE_** link in the **_README_** of a repository to clearly communicate the terms under which the code is shared and to establish legal permissions and limitations for potential users and contributors. 😃 👍